applications/TrilinosApplication/custom_strategies/builder_and_solvers/trilinos_block_builder_and_solver.h

```diff
@@ -510,10 +505,12 @@ class TrilinosBlockBuilderAndSolver
     {
         KRATOS_TRY
 
+        KRATOS_INFO_IF("TrilinosBlockBuilderAndSolver", BaseType::GetEchoLevel() > 2) << "Setting up the dofs" << std::endl;
+
         using DofsVectorType = Element::DofsVectorType;
-        
+
         // Gets the array of elements from the modeler
-        DofsVectorType dof_list;
+        DofsVectorType dof_list, second_dof_list;
```

Comment:
why do you need a `second_dof_list`? Is it used anywhere?

Reply:
In this case not, but will be use with the MPC, and this avoid conflicts, but I can remove it.